packages/nuxt-locokit/src/pages/admin/index.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <WithAsideNav>
+    <template #mini-navigation-items>
+      <NuxtLink
+        class="nav-link bg-primary text-white"
+        :to="{ name: ROUTES_NAMES.ADMIN.USERS.HOME }"
+      >
+        <button
+          v-tooltip="$t('pages.admin.users')"
+          type="button"
+          class="select-none mx-auto h-12 w-full focus:outline-none focus:ring-1 focus:ring-inset focus:ring-primary hover:bg-primary-dark p-2"
+        >
+          <i class="bi bi-person-fill" aria-hidden="true" />
+        </button>
+      </NuxtLink>
+      <NuxtLink
+        class="nav-link bg-primary text-white"
+        :to="{ name: ROUTES_NAMES.HOME }"
```

Comment:
this is a work in progress, and will be updated when the group page will be implemented ?